app/serializers/provider_serializer.rb

```diff
@@ -69,4 +70,41 @@ def postcode
   def region_code
     "%02d" % object.address_info['region_code'] if object.address_info['region_code'].present?
   end
+
+  def utt_application_alerts
+    # Temporarily create a value for this field which will be consistent
+    # for this provider. Remove this when we've data for this value to the
+    # db.
+    values = [
+      'No, not required',
+      'Yes, required',
+      'Yes - only my programmes',
+      'Yes - for accredited programmes only',
+    ]
+
+    select_value_for_provider(@object.provider_code, values)
+  end
+
+  def type_of_gt12
+    # Temporarily create a value for this field which will be consistent
+    # for this provider. Remove this when we've data for this value to the
+    # db.
+    values = [
+      'Coming / Enrol',
+      'Coming or Not',
+      'No response',
+      'Not coming',
+    ]
+
+    select_value_for_provider(@object.provider_code, values)
+  end
+
+private
+
+  def select_value_for_provider(provider_code, values)
```

Comment:
If we're just grabbing a random value, would it be better to call this `select_random_value_for_provider` to be explicit about that?

Reply:
I agree naming could be better, but this is temporary anyway, so will go away (soon?!).